animations/src/main/java/com/chiralbehaviors/CoRE/workspace/dsl/WorkspaceImporter.java

```diff
@@ -178,141 +177,190 @@ public void setScope(WorkspaceScope scope) {
         this.scope = scope;
     }
 
-    private void classifyAgencyAttributes() {
-        for (ClassifiedAttributeContext classified : wsp.getAgencyAttributeClassifications()) {
+    /**
+     * @param facet
+     */
+    private void agencyAuthorizations(FacetContext facet) {
+        // TODO Auto-generated method stub
```

Comment:
Yes.  I still have to add the cross existential entity network authorizations, and I wanted to leave a TODO so I could remember to do it.